odoo/addons/base/models/res_currency.py

```diff
@@ -134,11 +134,7 @@ def round(self, amount):
            :param float amount: the amount to round
            :return: rounded float
         """
-        # TODO: Need to check why it calls round() from sale.py, _amount_all() with *No* ID after below commits,
-        # https://github.com/odoo/odoo/commit/36ee1ad813204dcb91e9f5f20d746dff6f080ac2
-        # https://github.com/odoo/odoo/commit/0b6058c585d7d9a57bd7581b8211f20fca3ec3f7
-        # Removing self.ensure_one() will make few test cases to break of modules event_sale, sale_mrp and stock_dropshipping.
-        #self.ensure_one()
+        self.ensure_one()
```

Comment:
then, shouldn't the @api.multi tag on `round`, `compare_amounts` and `is_zero` methods be removed?

Reply:
No, `@api.multi` is the default anyway. It's meant to distinguish from `@api.model`, it's not a contradiction with `ensure_one()`.